### DISCUSSION TITLE
Windows 2000 now run on v86, so can we build some exe runner online using v86 and windows 2000?

### DISCUSSION THREAD

**Discussion by raspiduino** [copy/v86#499] [Ideas]
From some comments in the Issue, they said that they have run Windows 7 on v86. So I have an idea. We can run the image online, and then build some think to choose the exe file, run it in Windows 7 in v86, and open new Windows if needed. That must be awesome!

P/s: If I remember well, some one does have an idea about doing this but with Linux.

**Comment by copy** [comment]
[Windows XP or later versions aren't abandonware](https://forum.winworldpc.com/discussion/9882/rule-do-not-request-windows-xp-or-office-2000), so it's not very like to happen any time soon.

**Reply by raspiduino** [reply]
I saw Windows 2000 on v86 website. So instead of Windows 7, could we do the same thing with Windows 2000?

**Reply by raspiduino** [reply]
Or ReactOS?

**Reply by BelleNottelling** [reply]
I've been putting together a minimized image of ReactOS. If I remember correctly the final image size after an install was less than 150MB.
There's still more shrinkage that can be done, but I haven't done it yet.

I think that image would fit your idea well. The hard part is actually building out the system to upload your software into the VM

Also, ReactOS has some GDI/WIN32 issues that causes slow drawing and updates of windows

**Reply by Pixelsuft** [reply]
Maybe, use wine (Boxedwine) emscripten port?
For example: http://www.boxedwine.org/demo/

**Reply by raspiduino** [reply]
I will make one when I have time (with ReactOS/Windows2k)

**Reply by UserArham** [reply]
Windows XP or later models may happen soon, as this is an sucessor to WIndows 2000, does not mean you needed an IMG file.

**Comment by hello-smile6** [comment]
Wow! I'd love that!

**Reply by hello-smile6** [reply]
Let's make WSL Web!

**Comment by hello-smile6** [comment]
Windows 1 source code is public...

**Comment by UserArham** [comment]
Likely, you can run Windows XP on V86, as this take time, so to run it, do these steps. 1. Get an ISO file, 2. Create empty harddisk, and 3. Complete the setup.
